src/components/ThoughtsContainer.css

```diff
@@ -0,0 +1,155 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --font-size-standard: 16px;
+}
+
+body {
+  font-size: var(--font-size-standard);
+}
+.thoughts-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  border: 1px solid #ffadad;
```

Comment:
Is this border supposed to be deleted in the finished version? Cute with a little pink, though :)